botocore/handlers.py

```diff
@@ -556,8 +562,13 @@ def change_get_to_post(request, **kwargs):
      AutoPopulatedParam('accountId', 'Note: this parameter is set to "-" by \
                          default if no value is not specified.')
      .document_auto_populated_param),
-    ('docs.*.glacier.*.complete-section',
+    ('docs.*.glacier.UploadArchive.complete-section',
+     AutoPopulatedParam('checksum').document_auto_populated_param),
+    ('docs.*.glacier.UploadMultipartPart.complete-section',
      AutoPopulatedParam('checksum').document_auto_populated_param),
+    ('docs.request-params.glacier.CompleteMultipartUpload.complete-section',
+     document_glacier_tree_hash_checksum()),
```

Comment:
Why are you calling this function when registering the handler? That is probably why we are seeing leak test issues.